health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -33,13 +54,59 @@ const CampaignHome = () => {
           }}
         >
           <NewWindow />
           <div className={"descStyle"}>{t("HCM_CREATE_NEW_CAMPAIGN")}</div>
         </div>
       </div>
+      {showPopUp && (
+        <PopUp
+          // className={"boundaries-pop-module"}
+          type={"default"}
+          heading={t("CAMPAIGN_REQUIREMENTS_HEADING")}
+          children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)